src/app/components/WorkDetail/WorkDetail.jsx

```diff
@@ -27,31 +27,33 @@ class WorkDetail extends React.Component {
    * @param {object} work
    * @return {string|null}
    */
-  workDetailsObject(work) {
+  workDetailsObject() {
+    const { work } = this.props;
     return Object.keys(work).map(key => (
       [key, work[key]]
     ));
   }
 
-  /**
-   * onClick handler for resetting state for the request back to the home page
-   * to return the user to a new search.
-   *
-   * @param {object} event
-   */
-  handleReset(event) {
-    event.preventDefault();
-
-    this.boundActions.resetSearch();
-    this.context.router.push('/');
-  }
 
   render() {
     if (!this.props.work && _isEmpty(this.props.work)) {
       throw new Error('Work prop is missing or empty');
     }
     const { work } = this.props;
 
+    /**
```

Comment:
The indentation for this next block of comments is off.

Reply:
fixed